.github/workflows/ci.yml

```diff
@@ -225,3 +225,13 @@ jobs:
         path: |
           code-coverage-results.md
           pull_request_id
+
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      # https://github.com/marketplace/actions/pr-labeler-based-on-multiple-rules
```

Comment:
Why not use the [GitHub-provided labeler action](https://github.com/actions/labeler)?